Source/WebGPU/WGSL/Metal/MetalFunctionWriter.cpp

```diff
@@ -515,14 +515,11 @@ void FunctionDefinitionWriter::emitNecessaryHelpers()
     }
 
     if (m_shaderModule.usesMin()) {
-        m_stringBuilder.append(m_indent, "template<typename T>\n"_s,
```

Comment:
we can leave this templated if preferred, but I don't see it used outside of `uint`, can you confirm @tadeuzagallo ?